src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
```

Comment:
Grammar: Remove ending preposition.

How about, 
a) "We built a website for you to test your CSS knowledge."
b) "We created a website where you can practice your CSS styling."
c) "ARRR! Use yer CSS skills ter make yonder website shipshape."
d) Other... Just don't end with a preposition.

Reply:
Done! Thanks!